frontend/src/services/AccountAPI.ts

```diff
@@ -12,3 +12,11 @@ export const createBusinessAccount = async (
 ): Promise<AxiosResponse<any>> => {
   return axios.post(`accounts/business`, businessAccountRequestDTO);
 };
+
+export const getAllEmployeeAccounts = async (): Promise<any> => {
```

Comment:
You can specify the data type of your promise. It should be a `AxiosResponse<any>`. `Promise<AxiosResponse<any>>`